span-stacktrace/README.md

```diff
@@ -6,51 +6,23 @@ This module provides a `SpanProcessor` that captures the [`code.stacktrace`](htt
 Capturing the stack trace is an expensive operation and does not provide any value on short-lived spans.
 As a consequence it should only be used when the span duration is known, thus on span end.
 
-However, the current SDK API does not allow to modify span attributes on span end, so we have to
-introduce other components to make it work as expected.
+## Usage and configuration
 
-## Usage
+This extension supports autoconfiguration.
```

Comment:
maybe worth explaining this a bit more, e.g. explaining that just by having including it as a dependency, the Java SDK autoconfiguration module will automatically find and enable it

Reply:
Good point, I've enhanced the autoconfiguration description in 8745a7f